### ISSUE TITLE
export fetch?

### ISSUE THREAD

**Issue by wheresrhys** [matthew-andrews/isomorphic-fetch#11]
So that the slightly difficult to identify mistake of

```
var fetch = require('isomorphic-fetch')
```

doesn't happen

**Comment by wheresrhys** [comment]
or even better, export

```
module.exports = function () {
    console.log('you shouldn't assign fetch to a variable - it will polyfill globally')
}
```

**Comment by matthew-andrews** [comment]
but then it wouldn't be isomorphic anymore …

**Comment by wheresrhys** [comment]
how so?

**Comment by matthew-andrews** [comment]
the node javascript would differ from the client one?

**Comment by wheresrhys** [comment]
It could be implemented in client too couldn't it? The behaviour of the polyfill when incorrectly 'installed' isn't covered by the fetch spec, surely?

**Comment by matthew-andrews** [comment]
ok.  released in `v1.7.0`

**Comment by matthew-andrews** [comment]
thanks for reminding me about the fact I don't have to match the spec :-)
